File: Typescript/Practice/26072022/medium/medium_objectsToArrays.py
#!/usr/bin/env python

def to_list(dictionary):
  lst = []
  for key, value in dictionary.items():
    lst.append([key, value])
  return lst

print(to_list({ "a": 1, "b": 2 }))
print(to_list({ "shrimp": 15, "tots": 12 }))
print(to_list({}))

File: Typescript/Practice/medium/medium_objectsToArrays.ts
#!/usr/bin/env npx ts-node

function toArray (obj: { [key: string]: number }) {
  type array1 = (string|number)[];
  let arr: Array<array1> = [];
  for (let [key, value] of Object.entries(obj)) {
    arr.push([key, value]);
  };
  return arr;
};

console.log(toArray({ a: 1, b: 2 }));
console.log(toArray({ shrimp: 15, tots: 12 }));
console.log(toArray({}));

